edgelet/iotedge/src/check/checks/container_engine_is_moby.rs

```diff
@@ -0,0 +1,81 @@
+use failure::{self, Context, Fail};
+
+use crate::check::{checker::Checker, Check, CheckResult};
+
+#[derive(Default, serde_derive::Serialize)]
+pub struct ContainerEngineIsMoby {
+    docker_server_version: Option<String>,
+    #[cfg(windows)]
```

Comment:
Don't make this `#[cfg(windows)]`. Just let it be `None` so that the shape of the JSON is the same regardless of OS.